pom.xml

```diff
@@ -335,7 +335,7 @@
                 <plugin>
                     <groupId>community.flock</groupId>
                     <artifactId>graphql-simple-bindings-maven-plugin</artifactId>
-                    <version>0.6.1</version>
+                    <version>0.8.0</version>
```

Comment:
With `graphql-simple-bindings-maven-plugin` version 0.8.0 the package name is suffixed with a `.kotlin` package. As a result all references to classes generated by the plugin had their import changed. 


[example](https://github.com/flock-community/flock-eco/pull/94/files#diff-f8d3ec49cf7e95aa3176b057118e6648e5de4de59ea829e504c3c5797f25b654R3):

 ```diff
-import community.flock.eco.iso.language.graphql.Language
+import community.flock.eco.iso.language.graphql.kotlin.Language
```